test/lint/lint-python-dead-code-whitelist

```diff
@@ -0,0 +1,45 @@
+BadInputOutpointIndex  # unused class (test/functional/data/invalid_txs.py)
```

Comment:
nit: sort

Reply:
It is sorted, according to `cat ./*whitelist|sort` on my system

Reply:
Fascinating:

```
→ cat test/lint/lint-python-dead-code-whitelist | LC_ALL=en_US.UTF-8 sort
BadInputOutpointIndex  # unused class (test/functional/data/invalid_txs.py)
_.carbon_path  # unused attribute (contrib/macdeploy/custom_dsstore.py)
connection_lost  # unused function (test/functional/test_framework/mininode.py)
connection_made  # unused function (test/functional/test_framework/mininode.py)
[...]
→ cat test/lint/lint-python-dead-code-whitelist | LC_ALL=C sort
BadInputOutpointIndex  # unused class (test/functional/data/invalid_txs.py)
DuplicateInput  # unused class (test/functional/data/invalid_txs.py)
InvalidOPIFConstruction  # unused class (test/functional/data/invalid_txs.py)
NonexistentInput  # unused class (test/functional/data/invalid_txs.py)
OutputMissing  # unused class (test/functional/data/invalid_txs.py)
SizeTooSmall  # unused class (test/functional/data/invalid_txs.py)
SpendNegative  # unused class (test/functional/data/invalid_txs.py)
SpendTooMuch  # unused class (test/functional/data/invalid_txs.py)
TooManySigops  # unused class (test/functional/data/invalid_txs.py)
_.carbon_path  # unused attribute (contrib/macdeploy/custom_dsstore.py)
_.converter  # unused attribute (test/functional/test_framework/test_framework.py)
[...]
```